### PR TITLE
Updates source control standards

### PR DESCRIPTION
**Pull request by DannyGB**
A rejig of the source control standards, bringing it into one place and adding a template of an effective ReadMe

### CODE REVIEW

File: standards/source-control.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+> Any work created outside of the Skills Funding Agency organisation should be transferred into the Skills Funding Agency organisation at the earliest opportunity. [Guide to transferring a repository](https://help.github.com/en/articles/transferring-a-repository).
+
+It is accepted practice to access the SkillsFundingAgency GitHub organisation via personal GitHub accounts but before you can contribute to repositories in this organisation it is necessary to be granted the appropriate security role.
+
+## Secrets and Sensitive Configuration
+
+To ensure the safety and security of our systems; secrets, sensitive configuration and artefacts should not be pushed to public repositories. However this does not mean that private repositories are a good way to protect secrets, they are not. Secrets should be stored using technologies specifically designed for the task such as Azure Key Vaults, Secure Azure DevOps pipeline secret variables, etc...
+
```
Comment (benagain) [inline standards/source-control.md:23]: During development, settings files that are ignored by git - i.e. `appsettings.Development.json` - may be a convenient way to store secrets without them being accidentally added to the repository

**Review by benagain — APPROVED** [review]

File: standards/source-control.md (hunk 1 above)
Comment (DannyGB) [reply standards/source-control.md:23]: Happy to add this in

File: standards/source-control.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+project.
+
+## Creating an Effective Readme
+
+An effective and comprehensive Readme within a repository can help to simplify and accelerate the job new contributors have getting up to speed with a new code-base.
+
+An effective Readme should contain at least the following information:
+
+- An overview of the purpose of the code
+- All pre-requisites and dependencies will be identified
```
Comment (CodeWolf86) [inline standards/source-control.md]: > Will be identified

 to
> have been identified and listed   ?

 change of tense

File: standards/source-control.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+### 3. Delete branches when they are no-longer needed
+Once a branch is no longer needed it should be deleted, this helps keep the repository tidy and prevents the repository becoming littered with dead branches.
+
+[The GDS Way - Branching and Merging](https://gds-way.cloudapps.digital/standards/source-code.html#branching-merging-conventions)
+
+## Licensing
+
+Repositories should be [clearly named]({{ '/standards/naming-things' | relative_url }}),
+and have an [appropriate licence]({{ '/standards/licencing-software-or-code' | relative_url }})
+and enough documentation that someone new can get started with the
```
Comment (CodeWolf86) [inline standards/source-control.md]: Line break too early for >project?

File: standards/source-control.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+## Secrets and Sensitive Configuration
+
+To ensure the safety and security of our systems; secrets, sensitive configuration and artefacts should not be pushed to public repositories. However this does not mean that private repositories are a good way to protect secrets, they are not. Secrets should be stored using technologies specifically designed for the task such as Azure Key Vaults, Secure Azure DevOps pipeline secret variables, etc... 
+
+During development, settings files that are ignored by git - i.e. appsettings.Development.json - may be a convenient way to store secrets without them being accidentally added to the repository.
+
+## Code Branching
+
+Maintaining a consistent code branching strategy can help to ensure the development process is as smooth as possible and avoid many pitfalls when a team is  working on the same code-base.
```
Comment (CodeWolf86) [inline standards/source-control.md]: > is working
has a double space

**Review by CodeWolf86 — APPROVED** [review]

**Review by satya-sanikommu — APPROVED** [review]

**Pull request merged**